### PR TITLE
Mu 23

### PR DESCRIPTION
**Pull request by Staroz**
Please, review this PR.

### CODE REVIEW

File: cypress/e2e/updateCards.cy.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe("Test for update information of cards in Trello", function() {
             cy.addLabels(this.credentials.cardsNameArray[0], this.credentials.labelColor.green);
             cy.get('[data-testid="card-label"]').should('have.attr', 'data-color', this.credentials.labelColor.green);
         });
-        it("add Checklist", function () {
+        it.only("add Checklist", function () {
```
Comment (sphinix27) [inline cypress/e2e/updateCards.cy.js]: Remove the only

**Review by sphinix27 — CHANGES_REQUESTED** [review]
Make the changes and I'll approve.

**Comment by sphinix27** [comment]
Looks good but pipeline is failing, locators and timeouts seems to be the issue, please take a look at that.

File: cypress/support/commands.js [hunk 2]
```diff
@@ -19,21 +19,19 @@ Cypress.Commands.add('login', (email, pw, userName) => {
...
     // I did triple-click for a bug in this button, the "force:true" method does not work.
     cy.get('[data-testid="header-create-menu-button"]').dblclick().click();
     cy.get('[data-testid="header-create-team-button"]').click();
-    cy.get('[data-testid="header-create-team-name-input"]').type(workSpaceName)
-    cy.get('[class="t3Ou6F9HZxP3VK css-ufz0vj-control"]').click(); 
-    cy.contains('Education').click({force: true});
-    cy.wait(1500);
-    cy.get('[data-testid="header-create-team-submit-button"]').click();
+    cy.get('[data-testid="header-create-team-name-input"]').should('be.visible').type(workSpaceName)
+    cy.get('[class=" css-191o3mb"]').click();
```
Comment (sphinix27) [inline cypress/support/commands.js]: This locator can be rewritten as `'.css-191o3mb'`

File: cypress/support/commands.js [hunk 3]
```diff
@@ -131,7 +129,8 @@ Cypress.Commands.add('addLabels', (cardName, labelColor) => {
 Cypress.Commands.add('addChecklists', (cardName, checklistName) => {
     cy.get('[class="list-card js-member-droppable ui-droppable"]').contains(cardName).click();
     cy.get('[class="js-sidebar-action-text"]').contains('Checklist').click();
-    cy.get('[id="id-checklist"]').type(checklistName+'{enter}');
+    cy.get('[id="id-checklist"]').invoke('val', checklistName);
```
Comment (sphinix27) [inline cypress/support/commands.js]: Id locators are used like: `'#id-checklist'`

**Review by sphinix27 — APPROVED** [review]
Overall looks good, just read the comments and try to use those locators as suggested.

**Comment by Staroz** [comment]
Please, Review it  I had problems with the playwright's new version and the change of locators.

**Comment by Staroz** [comment]
Because some locators and functionality were chanched In Trello

**Pull request merged**